Firmware/MotorControl/controller.cpp

```diff
@@ -153,29 +153,29 @@ bool Controller::update(float* current_setpoint_output) {
         case INPUT_MODE_PASSTHROUGH: {
             pos_setpoint_ = input_pos_;
             vel_setpoint_ = input_vel_;
-            current_setpoint_ = input_current_;
+            torque_setpoint_ = input_torque_; //
         } break;
         case INPUT_MODE_VEL_RAMP: {
             float max_step_size = std::abs(current_meas_period * config_.vel_ramp_rate);
             float full_step = input_vel_ - vel_setpoint_;
             float step = std::clamp(full_step, -max_step_size, max_step_size);
 
             vel_setpoint_ += step;
-            current_setpoint_ = (step / current_meas_period) * config_.inertia;
+            torque_setpoint_ = (step / current_meas_period) * config_.inertia;
         } break;
         case INPUT_MODE_CURRENT_RAMP: {
```

Comment:
We should also rename this to Torque ramp.

Reply:
I think you meant this in reference to INPUT_MODE_CURRENT_RAMP. I changed that to INPUT_MODE_TORQUE_RAMP.